deploy/README.md

```diff
@@ -2,8 +2,8 @@
 
 Stratos can be deployed in the following environments:
 
-1. Cloud Foundry, as an application. See [guide](cloud-foundry)
-2. Kubernetes, using a Helm chart. See [guide](kubernetes)
-3. Docker, using docker compose. See [guide](docker-compose)
-4. Docker, single container deploying all components. See [guide](all-in-one)
+1. Cloud Foundry, as an application. See [guide](cloud-foundry/README)
```

Comment:
I don't think these links work with the added README - what was the reason for change? We have the same links in the main README as well.